### PR TITLE
feat: player physics simulate list shuffle

### PR DESCRIPTION
**Pull request by Kxnrl**
This pr fix entity touch order,
make trigger_ pick random first entity possible.

i also import default clang-format.
i make pr for cs2fixes feeling pain everytime.
cs2fixes code using tab indent, but it's bad to displaying for github review.
feel free to change to space.

### CODE REVIEW

File: .clang-format [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+Standard: Auto
+
+TabWidth: 4
+
+UseTab: Always
+
+SortIncludes: true
+
+IndentPPDirectives: AfterHash
```
Comment (xen-000) [inline .clang-format:143]: Personally I'm all for this, but reformatting the code will introduce a ton of merge conflicts to forks. I think @Vauff will be a better judge for how badly it'll affect his (GFL's) fork.


I also noticed some files were almost entirely run through clangformat which makes this hard to review, if we're doing this it should be in a separate commit/PR.

**Review by Vauff — APPROVED** [review]
